.buildkite/auditbeat/scripts/unit-tests-win.ps1

```diff
@@ -0,0 +1,51 @@
+$ErrorActionPreference = "Stop" # set -e
+$GoVersion = $env:GOLANG_VERSION # If Choco doesn't have the version specified in .go-version file, should be changed manually
+
+# Forcing to checkout again all the files with a correct autocrlf.
+# Doing this here because we cannot set git clone options before.
+function fixCRLF() {
+    Write-Host "-- Fixing CRLF in git checkout --"
+    git config core.autocrlf false
+    git rm --quiet --cached -r .
+    git reset --quiet --hard
+}
+
+function withGolang() {
+    Write-Host "-- Install golang $GoVersion --"
```

Comment:
Since we are on BK, maybe `--- Install golang $GoVersion` if we'd like it to be in a dedicated section?

Reply:
fixed